app/Http/Controllers/ProjectController.php

```diff
@@ -3,37 +3,82 @@
 namespace App\Http\Controllers;
 
 use App\Project;
+use App\Tag;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\Rule;
 
-class ProjectController extends Controller {
+class ProjectController extends Controller
+{
+	const PROJECTS = 'projects';
+	const SKILLSETS = 'skillsets';
+	const PRESENT = 'present';
+	const CATEGORY = 'category';
 
 	public function __construct() {
 		$this->middleware('auth');
 	}
 
 	public function index() {
 		$projects = Project::all();
-		return view('projects', compact('projects'));
+		return view(self::PROJECTS, compact(self::PROJECTS));
 	}
 
 	public function show($id) {
 		return view('projects.show', ['project' => Project::findOrFail($id)]);
 	}
 
 	public function create() {
-		return view('projects.create');
+		return view(
+			'registerProject', [
+				'courses' => Auth::user()->EnrolledIn,
+				'categories' => Tag::where('type', 2)->get(),
+				self::SKILLSETS => Tag::where('type', 1)->get()
+			]
+		);
 	}
 
 	public function store(Request $request) {
 		$attributes = request()->validate([
-			'name' => ['required'],
-			'video' => ['present'],
-			'long_description' => ['present'],
-			'short_description' => ['present'],
-			'photo' => ['present']
+			'projectName' => ['required'],
+			'videoPitch' => [self::PRESENT],
+			'longDescription' => [self::PRESENT],
+			'shortDescription' => [self::PRESENT],
+			'projectImage' => [self::PRESENT],
+			self::CATEGORY => [
+				self::PRESENT,
+				Rule::in(Tag::where('type', 2)->pluck('id'))
+			],
+			self::SKILLSETS => 'present|array|min:1',
+			'skillsets.*' => [
+				'integer',
+				Rule::in(Tag::where('type', 1)->pluck('id')),
+			]
 		]);
 
-		$project = Project::create($attributes);
-		return view('projects.show', ['project' => $project]);
+		$project = $this->saveRecord($attributes);
+		if (!isset($project)) {
```

Comment:
`create` is always set (regresa una instancia del Modelo) so you wanna do `if($project->exists)`